dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/export/enrollment/EnrollmentsExportController.java

```diff
@@ -70,11 +71,13 @@
 {
     protected static final String ENROLLMENTS = "enrollments";
 
-    private static final String DEFAULT_FIELDS_PARAM = "*,!relationships,!events,!attributes";
+    private static final String DEFAULT_FIELDS_PARAM_STRING = "*,!relationships,!events,!attributes";
```

Comment:
Can `DEFAULT_FIELDS_PARAM_STRING` and `DEFAULT_FIELDS_PARAMS` live in `RequestParams`? This would make perfect sense for me in terms of where they belong.